ts/handler_utils/torch_export/load_model.py

```diff
@@ -0,0 +1,27 @@
+import tempfile
+
+import torch.fx._pytree as fx_pytree
+from torch._inductor.utils import aot_inductor_launcher, cache_dir
+from torch.utils import _pytree as pytree
+from torch.utils.cpp_extension import load_inline
+
+
+def load_exported_model(model_so_path, device):
+    module = load_inline(
+        name="aot_inductor",
```

Comment:
would we need to change the name in the case of multiple workers?

Reply:
tried with multiple workers. Shared a screenshot of nvidia-smi in the PR